.landscaper/gardenlet/README.md

```diff
@@ -0,0 +1,91 @@
+# Gardenlet Landscaper Component
+
+This component is supposed to be used in the context of the [Gardener Landscaper](https://github.com/gardener/landscaper).
+Its purpose is to deploy a Gardenlet to a Kubernetes cluster
+and automatically register this cluster as a `Seed` resource in the Gardener installation.
+
+TLS bootstrapping using a bootstrap token is used to obtain a valid Gardenlet certificate for the Garden cluster.
+Essentially follows this [documentation](../../docs/deployment/deploy_gardenlet_manually.md).
+
+## Run
+
+The Gardenlet landscaper is supposed to run as part of the [Gardener Landscaper](https://github.com/gardener/landscaper) but can also be executed stand-alone.
+
+The Gardenlet landscaper does not support command line arguments.
+However, make sure to set the following environment variables
+- `IMPORTS_PATH` contains the path to a file containing [required import configuration](#required-configuration).
+- `OPERATION` is set to either `RECONCILE` (deploys the Gardenlet) or `DELETE` to remove the deployed resources.
+- `COMPONENT_DESCRIPTOR_PATH` contains the path to a file containing a component descriptor for the Gardenlet. 
+   This file contains the OCI image reference to use for the Gardenlet deployment.
+   You can find a sample descriptor [here](component_descriptor_list.yaml).
+   
+The Gardenlet landscaper can be run locally by executing the below `make` statement.
+The filepath to a valid landscaper import configuration file has to be provided as the first argument.
+The [example file](./example/imports.yaml) can be used as a blueprint.
+
+```
+make start-landscaper-gardenlet IMPORT_PATH=<filepath>
+```
+
+## Import Configuration
+
+You can find an example import configuration file [here](./example/imports.yaml).
+
+### Required configuration
+
+- `runtimeCluster` contains the kubeconfig for the Kubernetes cluster where the Gardenlet is deployed to. 
+   Requires admin permissions. Only used for the Gardenlet deployment. 
+   This is the Kubernetes cluster targeted as Seed (via in-cluster mounted service account token),
+   if not otherwise specified in `.componentConfiguration.seedClientConnection.kubeconfig`.
```

Comment:
What does

> (via in-cluster mounted service account token), if not otherwise specified in `.componentConfiguration.seedClientConnection.kubeconfig`.

mean? Can you improve this sentence?